README.md

```diff
@@ -114,11 +114,10 @@ documentation of the latest stable version.
 
 ### Verifying Binaries
 
-Current, LTS, and Nightly download directories all contain a SHASUMS256.txt
-file that lists the SHA checksums for each file available for
-download.
+Download directories contain a SHASUMS256.txt file with SHA checksums for the
```

Comment:
`SHASUMS256.txt` -> `` `SHASUMS256.txt` `` (with backticks)?

(Here and in all other mentions of `SHASUMS256.txt` and `SHASUMS256.txt.sig` if it is appropriate for this PR).